plugins/modules/api_info.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: api_info
+
+author:
+  - Tjaž Eržen (@tjazsch)
+
+short_description: Manage ServiceNow GET requests
+description:
+  - Retrieve records via ServiceNow REST Table API for an arbitrary table.
+  - For more information, refer to the ServiceNow REST TAble API documentation at
+    U(https://docs.servicenow.com/bundle/paris-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html#c_TableAPIO).
+version_added: 2.0.0
+extends_documentation_fragment:
+  - servicenow.itsm.instance
+  - servicenow.itsm.sys_id.info
+seealso:
+  - module: servicenow.itsm.api
+options:
+  resource:
+    description:
+      - The name of the table that we want to obtain records from
+    type: str
+    required: true
+  sysparm_query:
+    description:
+      - An encoded query string used to filter the results.
+      - List of all possible operators and a guide on how to map them to form a query may be found at
+        U(https://docs.servicenow.com/en-US/bundle/sandiego-platform-user-interface/page/use/common-ui-elements/reference/r_OpAvailableFiltersQueries.html).
+        and U(https://developer.servicenow.com/dev.do#!/reference/api/sandiego/rest/c_TableAPI) under 'sysparm_query'.
+    type: str
+  display_value:
+    description:
+      - "Return field display values (true), actual values (false), or both (all) (default: false)."
+    type: str
+    choices: ["true", "false", "both"]
+  exclude_reference_link:
+    description:
+      - "true to exclude Table API links for reference fields (default: false)"
```

Comment:
Please note that while ServiceNow's default is `false`, this collection's default is `true` (cf. `module_utils/table.py`, function `_query`). I would keep it as ServiceNow's default, but the code would need to be adapted to that.

Reply:
Because of the enforced value set to `true` in the above mentioned function `_query`, setting this parameter to either `false`, `true` or omitting it makes no difference to the resulting records. Since fixing this issue would require a change in function `_query` and could potentially affect the results of other modules if not enough care is taken, I would suggest to fix this bug in a separate pull request, which should be merged prior to this pull request.

Reply:
Ahh thanks for noticing that @uscinski! Created an issue, in which this is going to be resolved (https://github.com/ansible-collections/servicenow.itsm/issues/181)

Reply:
Pushed a seperate pull request to resolve this: https://github.com/ansible-collections/servicenow.itsm/issues/181.